src/components/header/_header.scss

```diff
@@ -108,6 +108,8 @@
     &:active {
       margin-bottom: -1px; // Negate transparent bottom border
       border-bottom: 1px solid currentColor;
```

Comment:
Out of curiosity, I think that border's default colour is `currentColor` which could mean changing this statement to `border-bottom: 1px solid` may remove the need for the conditional IE8 logic?

Reply:
Great suggestion – that seems to work well.